Microsoft/microsoft-intune/ingest/parser.yml

```diff
@@ -27,7 +27,7 @@ stages:
           host.mac: '{{json_event.message.properties.WifiMacAddress}}'
           host.name: '{{json_event.message.properties.ManagedDeviceName or json_event.message.properties.DeviceHostName }}'
           host.type: '{{json_event.message.properties.Model}}'
-          intune.compliant: '{{json_event.message.properties.CompliantState}}'
+          microsoft.intune.compliant: '{{json_event.message.properties.CompliantState}}'
```

Comment:
I think that in vendor fields, we should stay as close as possible from the original field name (here `properties.CompliantState`)

Maybe @squioc as a different point of view ?

Reply:
So, @pve you suggest `microsoft.intune.compliant_state`?